src/ruby/A47ReplaceWord.java

```diff
@@ -0,0 +1,29 @@
+package ruby;
+
+import java.util.Arrays;
+
+/*Replace second last occurance of "Hi" with your name.
+String msg = "Hi globant Hello Hi Hi globant Hi Hello Hello globant Hi Techno Credits Techno";
+output : Hi globant Hello Hi Hi globant Maulik Hello Hello globant Hi Techno Credits Techno   */
+
+public class A47ReplaceWord {
+	static void findWord(String input) {
+		System.out.println("Input String: " + input);
+		int count = 0;
+		String[] wordArr = input.split(" ");
+		for (int index = wordArr.length - 1; index >= 0; index--) {
+			if (wordArr[index].equalsIgnoreCase("Hi")) {
+				count++;
+				if (count == 2) {
```

Comment:
this is also good approach
also check this approach
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/1450/files